src/runtime/entitlements-manager-test.js

```diff
@@ -336,6 +341,34 @@ describes.realWin('EntitlementsManager', {}, (env) => {
       expect(ents.enablesThis()).to.be.false;
     });
 
+    it('adds dev mode params to request when present on the URL', async () => {
+      const scenario = 'TEST_SCENARIO';
+      win.location.hash = `#swg.debug=1&swg.deventitlement=${scenario}`;
+      xhrMock
+        .expects('fetch')
+        .withExactArgs(
+          '$frontend$/swg/_/api/v1/publication/pub1/entitlements?devEnt=' +
+            encodeURIComponent(scenario) +
+            '&crypt=' +
+            encodeURIComponent(encryptedDocumentKey),
+          {
+            method: 'GET',
+            headers: {'Accept': 'text/plain, application/json'},
+            credentials: 'include',
+          }
+        )
+        .returns(
+          Promise.resolve({
+            text: () => Promise.resolve('{}'),
+          })
+        );
+      expectGetSwgUserTokenToBeCalled();
+
+      const ents = await manager.getEntitlements({
```

Comment:
Looks like we can remove this var
```suggestion
      await manager.getEntitlements({
```